src/uu/chown/src/chown.rs

```diff
@@ -330,12 +318,15 @@ macro_rules! unwrap {
 }
 
 impl Chowner {
-    fn exec(&self) -> i32 {
+    fn exec(&self) -> UResult<()> {
         let mut ret = 0;
         for f in &self.files {
             ret |= self.traverse(f);
         }
-        ret
+        if ret != 0 {
+            return Err(UError::from(ret));
+        }
+        Ok(())
```

Comment:
This still feels awkward and made me realize that `traverse`, `obtain_meta` might also be nice to convert to `UResult`. You could even change `wrap_chown` if you want (although that is also used by `install`)